requirements.txt

```diff
@@ -1,5 +1,5 @@
 hail
 hdbscan
 ipywidgets
-sklearn
```

Comment:
The [sklearn project page](https://pypi.org/project/sklearn/) says "Use scikit-learn instead".

Reply:
We use `sklearn` in the code though - either we have to change that and make sure it works 1:1, or revert this for now and do that later. I leave it up to you

Reply:
Only the [distribution package](https://packaging.python.org/glossary/#term-distribution-package) has changed. The [import package](https://packaging.python.org/glossary/#term-import-package) is still `sklearn`.

https://scikit-learn.org/stable/getting_started.html

The only import from scikit-learn is `from sklearn.ensemble import RandomForestClassifier`, which works after pip installing `scikit-learn` instead of `sklearn`.